.travis.yml

```diff
@@ -51,7 +51,7 @@ git:
 jobs:
   include:
   - os: osx
-    osx_image: xcode10.1
+    osx_image: xcode10.2
```

Comment:
The dependency is the macOS version... I am not sure if there is a difference between 10.4 and 10.4.6... if there is none, we might want to update to xcode 11.3

Reply:
Updated.